mkdocs/docs/HPC/apptainer.md

```diff
@@ -47,11 +47,11 @@ filesystems, and they will be accessible via the familiar `$VSC_HOME`,
 ### Creating Apptainer/Singularity images
 
 Creating new Apptainer/Singularity images or converting Docker images,
-by default, requires admin privileges, which is obviously not available
+by default, requires admin privileges, which are not available
```

Comment:
I'm not sure this is still true with the latest version of Apptainer, so maybe we should rephrase this, and stop recommending `--fakeroot`.

@lbarraga Can you check on the status of this, and rework accordingly?

Reply:
The apptainer version used on the HPC is `1.3.3-1.el8`. Consulting the apptainer `1.3` docs it seems [--fakeroot is still an option](https://apptainer.org/docs/user/latest/build_a_container.html#fakeroot)

Reply:
The option is still supported, sure, but is it actually still *required* on our systems?

There were some enhancements made in Apptainer recently that may make that it's no longer required...

Reply:
@boegel Ok, so it does seem to work without `--fake-root`. but not without some small warnings:

```shell
[vsc46420@gligar08 ~]$ apptainer build /tmp/$USER/tf.sif docker://nvcr.io/nvidia/tensorflow:21.10-tf1-py3
INFO:    Starting build...
...
2024/09/13 12:10:45  warn rootless{usr/local/nvm/versions/node/v16.6.1/bin/npm} ignoring (usually) harmless EPERM on setxattr "user.rootlesscontainers"
2024/09/13 12:10:45  warn rootless{usr/local/nvm/versions/node/v16.6.1/bin/npx} ignoring (usually) harmless EPERM on setxattr "user.rootlesscontainers"
...
INFO:    Creating SIF file...
INFO:    Build complete: /tmp/vsc46420/tf.sif
```

Should I remove `--fake-root` from the docs?

Reply:
Yes, those warnings are harmless (as they suggest themselves)